gnd/src/main/java/com/google/android/gnd/ui/basemapselector/BasemapSelectorViewModel.java

```diff
@@ -0,0 +1,12 @@
+package com.google.android.gnd.ui.basemapselector;
+
+import androidx.lifecycle.ViewModel;
+
+import javax.inject.Inject;
+
+public class BasemapSelectorViewModel extends ViewModel {
```

Comment:
Please add basic Javadoc about what this represents (i.e. the state of the UI used for selecting basemap imagery for offline use).

Reply:
Done